ccstake/src/summerizer.ts

```diff
@@ -67,22 +81,29 @@ export async function summarize(sdk: SDK, blockNumber: number) {
         }
     }
     const accounts = Object.keys(aggregate).map(PlatformAddress.ensure);
-    return {
+    const summary: Summary = {
```

Comment:
I think you can just do `return {...` here. It'll still compare this value with the return type.

Reply:
It fixes summary type to `Summary` type.
Returning an object in the async function will cause TypeScript to make an inference that the object is `T | PromiseLike<T>`. `PromiseLike<T>` part will causes `this.get(...)` to fail with this error:
```
Property 'get' does not exist on type 'Summary | PromiseLike<Summary>'.
  Property 'get' does not exist on type 'PromiseLike<Summary>'.
```